syntax = "proto3";

message MessageWrapper {
    string type = 1;
    bytes payload = 2;
}